source/Introduction/How-SpaceROS-Differs.rst

```diff
@@ -1,22 +1,118 @@
 How Space ROS Differs from ROS 2
```

Comment:
Again this file might be renamed `How-Space-ROS-Differs.rst` to match the spelling conventions for Space ROS.

Reply:
Yes. Changed the filename.